### PR TITLE
chore: update quick-start

### PR DESCRIPTION
**Pull request by hardfist**

### CODE REVIEW

File: docs/zh/guide/quick-start.mdx [hunk 1]
```diff
@@ -1,9 +1,51 @@
...
 
 通过本章节来了解如何快速上手并使用 Rspack。
 
 ## 使用脚手架快速创建
 
+### 使用 Rsbuild
+
+我们推荐使用 [Rsbuild](#rsbuild) 创建新项目，调用如下命令即可：
+
+<Tabs values={[{ label: 'npm' }, { label: 'yarn' }, { label: 'pnpm' }, { label: 'bun'}]}>
```
Comment (chenjiahan) [inline docs/zh/guide/quick-start.mdx:13]: PackageManagerTabs provides better UI:

https://rspress.dev/api/client-api/api-components.html#packagemanagertabs

Comment (hardfist) [reply docs/zh/guide/quick-start.mdx]: fix

**Review by ScriptedAlchemy — APPROVED** [review]

File: docs/en/guide/quick-start.mdx [hunk 2]
```diff
@@ -1,46 +1,24 @@
...
-
-  </Tab>
-
-  <Tab>
+You can scaffold a simple project setup using [Rsbuild](#rsbuild) directly. Invoke the following command:
 
-```sh
-pnpm create rspack@latest
-```
+<PackageManagerTabs command="install rsbuild -D" />
```
Comment (chenjiahan) [inline docs/en/guide/quick-start.mdx:13]:
```suggestion
<PackageManagerTabs command="create rsbuild -D" />
```

File: docs/en/guide/quick-start.mdx [hunk 3]
```diff
@@ -1,46 +1,24 @@
...
+### Using the Rspack CLI
 
-```sh
-bun create rspack@latest
-```
+You can scaffold a simple project setup using the Rspack CLI directly. Invoke the following command:
 
-  </Tab>
-</Tabs>
+<PackageManagerTabs command="install rspack -D" />
```
Comment (chenjiahan) [inline docs/en/guide/quick-start.mdx:21]:
```suggestion
<PackageManagerTabs command="create rspack -D" />
```

File: docs/zh/guide/quick-start.mdx [hunk 4]
```diff
@@ -1,46 +1,24 @@
...
-你可以直接使用 Rspack CLI 来创建一个基础项目，调用以下命令即可：
+我们推荐使用 [Rsbuild](#rsbuild) 创建新项目，调用如下命令即可：
 
-<Tabs values={[{ label: 'npm' }, { label: 'yarn' }, { label: 'pnpm' }, {label: 'bun'}]}>
-  <Tab>
-
-```sh
-npm create rspack@latest
-```
+<PackageManagerTabs command="install rsbuild -D" />
```
Comment (chenjiahan) [inline docs/zh/guide/quick-start.mdx:13]:
```suggestion
<PackageManagerTabs command="create rsbuild@latest" />
```

File: docs/en/guide/quick-start.mdx (hunk 2 above)
Comment (hardfist) [reply docs/en/guide/quick-start.mdx:13]: any difference

File: docs/zh/guide/quick-start.mdx [hunk 5]
```diff
@@ -1,46 +1,24 @@
...
-```sh
-bun create rspack@latest
-```
+### 使用 Rspack CLI
 
-  </Tab>
-</Tabs>
+你可以直接使用 Rspack CLI 来创建一个基础项目，调用以下命令即可：
 
+<PackageManagerTabs command="install rspack -D" />
```
Comment (chenjiahan) [inline docs/zh/guide/quick-start.mdx:21]:
```suggestion
<PackageManagerTabs command="create rspack@latest" />
```

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**